Source/Scene/Expression.js

```diff
@@ -541,27 +541,39 @@ define([
 
     function parseKeywordsAndVariables(ast) {
         if (isVariable(ast.name)) {
-            return new Node(ExpressionNodeType.VARIABLE, getPropertyName(ast.name));
+            var name = getPropertyName(ast.name);
+            if (name.substr(0, 8) === 'tiles3d_') {
+                return new Node(ExpressionNodeType.GLOBAL_VARIABLE, name);
```

Comment:
"GLOBAL" is the wrong semantic.  In GLSL in Cesium, these are like "built-in" constants or "automatic" uniforms.  Roll with whatever semantic you like for 3D Tiles.